ultralytics/yolo/engine/exporter.py

```diff
@@ -362,6 +362,16 @@ def export_openvino(self, prefix=colorstr('OpenVINO:')):
                                     model_name=self.pretty_name,
                                     framework='onnx',
                                     compress_to_fp16=self.args.half)  # export
+        ov_model.set_rt_info('YOLOv8', ['model_info', 'model_type'])
+        ov_model.set_rt_info('fit_to_window_letterbox', ['model_info', 'resize_type'])
```

Comment:
It seems like classification models have a different type of pre-processing. @Wovchena, can we handle it in this PR and place different settings based on the model type?

Reply:
Resolved in my latest commit with if model_type != classify